packages/manager/modules/config/src/environment/environment.service.js

```diff
@@ -1,11 +1,15 @@
+import { ALLOWED_REGIONS, DEFAULT_REGION } from './environment.constants';
+
 export default class EnvironmentService {
   constructor() {
-    this.region = 'EU';
+    this.region = DEFAULT_REGION;
     this.version = null;
   }
 
-  setRegion(region = 'EU') {
-    this.region = region;
+  setRegion(region = DEFAULT_REGION) {
+    if (ALLOWED_REGIONS.includes(region)) {
+      this.region = region;
```

Comment:
**thought**: Maybe we could throw an error or add a log if region is not supported